packages/@react-stately/slider/test/useSliderState.test.js

```diff
@@ -12,7 +28,8 @@ describe('useSliderState', () => {
       formatOptions: {
         style: 'currency',
         currency: 'USD'
```

Comment:
nit: can remove this now that `useSliderState` doesn't consume `formatOptions`

Reply:
Maybe we should move `formatOptions` just to the spectrum types since nothing uses them anymore?